src/main/java/setup/cardlists/MakeAllCardsList.java

```diff
@@ -14,173 +14,197 @@
+        return legalCards;
+    }
+
+    private static Map<String, Double> getCardNamesAndPrices(List<String> html)  {
+        Map<String, Double> cards = new HashMap<>();
+        int linesSinceMarker = -1;
+        String lastCardFound = null;
+
+        for (String str : html){
+
+            linesSinceMarker++;
+
+            if (str.length() > 17 && str.substring(0, 17).equals("<td class='card'>") && (str.indexOf("#online\">") > -1) || (str.indexOf("#paper\">") > -1)) { //Looks for a marker line
+
+                //Get the locations of the second ">" and the third "<", the card's name is located between them
+                int secondClose = str.indexOf('>',str.indexOf('>')+1);
+                int thirdOpen = str.indexOf('<',str.indexOf('<',str.indexOf('<')+1)+1);
+                String thisCard = str.substring(secondClose+1,thirdOpen);
+
+                //Lots of complicated formatting stuff. Don't worry about it.
+                String cardname = formatEnglishName(thisCard).replace("&#39;", "'").replace("Lim-Dul","Lim-Dûl").replace("Jotun","Jötun");
+
+                //MTGGoldfish doesn't include the accent, but it should, so I add it.
+                switch (cardname){
+                case "Seance": cardname = "Séance";
+                break;
+                case "Dandan": cardname = "Dandân";
+                break;
+                case "Khabal Ghoul": cardname = "Khabál Ghoul";
+                break;
+                case "Junun Efreet": cardname = "Junún Efreet";
+                break;
+                case "Ghazban Ogre": cardname = "Ghazbán Ogre";
+                break;
+                case "Ifh-Biff Efreet": cardname = "Ifh-Bíff Efreet";
+                break;
+                case "Ring of Ma'ruf": cardname = "Ring of Ma'rûf";
+                break;
+                }
+
+                // Retain knowledge of the last card we found
+                lastCardFound = cardname;
+
+                // Then start counting lines until we hit the proper one.
+                linesSinceMarker = 0;
+            }
+
+            // If we've found a card, then the third line after the marker has the card price
+            if (lastCardFound != null && linesSinceMarker == 4){
+                cards.put(lastCardFound, Double.parseDouble(str));
+
+                lastCardFound = null;
+                linesSinceMarker = -1;
+            }
+        }
+
+        return cards;
+    }
+
+    private static String formatEnglishName(String name){
+        String formatted = "";
+        for (int c = 0; c < name.length(); c++){
+            char letter = name.charAt(c);
+
+            //Some cards with multiple promo printings have (second promo) or something in parens, this gets rid of that so we don't get duplicates
+            if(letter == '('){
+                formatted = formatted.trim();
+                break;
+            }
+            else
+                formatted += letter;
+        }
+        return formatted;
+    }
```

Comment:
Why are all the lines changed here? I guess you're probably abandoning this PR now but if not could you resubmit with only the changed lines changed (tabs versus spaces?)

Reply:
ok